src/entities/Proposal/utils.ts

```diff
@@ -212,3 +214,11 @@ export const isCoauthor = async (proposalId: string, address: string): Promise<b
     .getCoAuthorsByProposal(proposalId, CoauthorStatus.APPROVED)
     .then((coauthors) => !!coauthors.find((coauthor) => coauthor.address === address))
 }
+
+const TRANSPARENCY_ONE_TIME_PAYMENT_TIERS = new Set(['Tier 1', 'Tier 2'])
+
+export function isProposalInCliffPeriod(grant: GrantAttributes) {
+  const isOneTimePayment = TRANSPARENCY_ONE_TIME_PAYMENT_TIERS.has(grant.configuration.tier)
+  const now = Time.utc()
+  return !isOneTimePayment && Time.unix(grant.enacted_at).add(CLIFF_PERIOD_IN_DAYS, 'day').isAfter(now)
```

Comment:
`const isOneTimePayment = TRANSPARENCY_ONE_TIME_PAYMENT_TIERS.has(grant.configuration.tier)`

This could be another question the object could know how to reply: `grant.isOneTimePayment()`

and `grant.isInCliffPeriod()`